qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOfferInput.groovy

```diff
@@ -9,4 +15,21 @@ package life.qbic.portal.portlet.offers.create
  */
 interface CreateOfferInput {
 
+
+    /**
+     * Creates a new quotation based on the given TOMATOID
+     * @param tomatoId makes it possible to identify the quotation
+     * @since 1.0.0
+     */
+    void createOffer(String tomatoId)
```

Comment:
Can this be a TomatoId object also? Is TomatoId a DTO?

Reply:
Yes, it is, this was still a leftover of our old architecture. I would even argue to delete the method. What do you think? Or do we really want to search for an offer based on its id? Which makes no sense in the createOffer use case imo

Reply:
Yes I also think that this is not the right place for it. Assuming we provide all the information at once and do not store incomplete offers int he DB. @sven1103 I do not know what behaviour is desired for the application here.